androidApp/src/main/java/com/projectnewm/di/NetworkModule.kt

```diff
@@ -0,0 +1,30 @@
+package com.projectnewm.di
```

Comment:
Should NetworkModule and UseCaseModule have `expect`s?

Reply:
See note above about Hilt